app/controllers/attendees_controller.rb

```diff
@@ -14,7 +14,7 @@ def create
     @attendee = @event.attendees.build(attendee_params)
 
     if @attendee.save
-      redirect_to Event.find(@attendee.event_id)
+      redirect_to events_path
```

Comment:
This is not making sense to redirect to `events_path`
Most of the time we would redirect user to that page from where he opened form;
For this, we can redirect to `event_attendees_path(@event)` as we are doing in `update`